src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,104 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { Loader } from '../Loader';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+
+export const PeopleTable: React.FC = () => {
+  const [peopleData, setPeopleData] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState('');
+
+  const { slug } = useParams();
+  const isSelect = peopleData.find(person => person.slug === slug);
+
+  const fetchPeopleData = async () => {
+    try {
+      const response = await fetch(
+        'https://mate-academy.github.io/react_people-table/api/people.json',
+      );
+
+      const data = await response.json();
+
+      setPeopleData(data);
+    } catch {
+      {
+        setError('Data loading error');
+      }
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    fetchPeopleData();
+  }, []);
+
+  console.log(peopleData);
```

Comment:
Consider removing the console.log statement before deploying the application to production, as it may clutter the console output.